skaffold.yaml

```diff
@@ -9,6 +9,9 @@
     serviceAccount:
       name: im-manager-prod
       namespace: instance-manager-prod
+#    setValueTemplates:
+#      corsAllowOrigins:
+#        - "https://{{ .UI_URL }}"
```

Comment:
Why is that commented out? I would only add commented out code with an explanation. I have no clue why this is here and if I can delete it or when I should uncomment it 🤷🏻